Geometry_Engine/Query/EndDir.cs

```diff
@@ -50,20 +57,40 @@ public static Vector EndDir(this Circle circle)
 
         /***************************************************/
 
+        [Description("Gets the tangent vector at the end of the Ellipse.")]
+        [Input("ellipse", "The Ellipse from which to get the tangent at the end point.")]
+        [Output("endTan", "The tangent vector at the end of the Ellipse.")]
+        public static Vector EndDir(this Ellipse ellipse)
+        {
+            //Ellipse starts and ends at the first axis point, and move towards the second axis, hence second axis is the tangent at start and end
```

Comment:
This could possibly be said in the description instead of a code comment

Reply:
Yeah, happy to move it up :)